cmd/bacalhau/docker_run.go

```diff
@@ -222,6 +215,14 @@ func init() { //nolint:gochecknoinits,funlen // Using init in cobra command is i
 		&ODR.IDOnly, "id-only", ODR.IDOnly, "Print out only the Job ID on successful submission.",
 	)
 
+	dockerRunCmd.PersistentFlags().BoolVar(
+		&ODR.RunTimeSettings.AutoDownloadResults,
+		"download",
+		ODR.RunTimeSettings.AutoDownloadResults,
+		"Should we download the results once the job is complete?",
+	)
```

Comment:
Why not add this to `setupRunTimeFlags`? (Disclousre: I've just spent time adding the common flags to WASM so if you don't I probably will :D)

Reply:
Great point :+1: